src/scripts.js

```diff
@@ -92,7 +92,7 @@ window.addEventListener("load", () => {
 });
 
 window.addEventListener('resize', () => {
-  renderGrid(pageData.recipesOfInterest)
+  renderGrid(pageData.allRecipes)
```

Comment:
This introduces a new bug by changing our page from its current state (recipes of interest) to all recipes. If you are in the your recipes view and resize your screen, then it will reset to all recipes in that view. Same for resizing after clicking on tags or searching